t/deprecation-warnings.t

```diff
@@ -0,0 +1,35 @@
+use lib 't';
+use TestAPIcastBlackbox 'no_plan';
+
+repeat_each(1);
+run_tests();
+
+__DATA__
+
+=== TEST 1: authentication credentials missing
```

Comment:
I'd change the title and description of the test so they make reference to what is actually being tested here.

Reply:
hah, yeah my copy paste issue
well spotted

Reply:
:eyes: